src/d3a/d3a_core/simulation.py

```diff
@@ -448,15 +423,14 @@ def _handle_input(self, console, sleep: float = 0):
         while True:
             cmd = console.get_char(timeout)
             if cmd:
-                if cmd not in {'i', 'p', 'q', 'r', 'S', 'R', 's', '+', '-'}:
+                if cmd not in {'i', 'p', 'q', 'r', 'R', 's', '+', '-'}:
                     log.critical("Invalid command. Valid commands:\n"
                                  "  [i] info\n"
                                  "  [p] pause\n"
                                  "  [q] quit\n"
                                  "  [r] reset\n"
-                                 "  [S] stop\n"
+                                 "  [s] stop\n"
                                  "  [R] start REPL\n"
-                                 "  [s] save state\n"
```

Comment:
Thanks a lot for this refactoring! I guess that it is not worth it to reuse the new current_state into a pickle file for this option? WE never used it right? So I agree to remove it.

Reply:
No worries, we never used pickle for saving the state of the simulation (it used to be part of our legacy code, was not working right from the start IIRC), and it is guaranteed to fail (Area and Market classes are not pickleable)